tailwind.config.js

```diff
@@ -8,7 +8,13 @@ module.exports = {
     './public/index.html',
   ],
   theme: {
-    extend: {}
+    colors: {
+      'midnightblue': '#2f2e7c',
+      'darkslateblue': '#234894',
+      'steelblue': '#3b85c1',
+      'slategray': '#8f5ba3',
+      'plum': '#cfa6ce',
+    },
```

Comment:
I think you need use extend because on this way you will ovewrite the all the default colors of tailwind.
```suggestion
    extend: {
      colors: {
        'midnightblue': '#2f2e7c',
        'darkslateblue': '#234894',
        'steelblue': '#3b85c1',
        'slategray': '#8f5ba3',
        'plum': '#cfa6ce',
      }
     },
```